hypothesis-python/tests/cover/test_statistical_events.py

```diff
@@ -252,6 +254,8 @@ def threshold(error):
     assert stats["targets"]["error"] > 10
 
 
+# describe_statistics causes not-deterministic crosshair errors for some reason?
+@xfail_on_crosshair(Why.other)
 def test_statistics_with_events_and_target():
```

Comment:
link to upstream issue report, so we know when we can remove this?  (and to remind use + phillip to fix it)

Reply:
I'm not sure this is crosshair's fault yet! I will spend 10 minutes investigating, and if I cannot figure it out I will give up and leave the xfail